packages/common/src/tools/queue/Retries.ts

```diff
@@ -42,4 +42,13 @@ export const Retries = {
   always,
   exponentialBackOff,
   maxAttempts,
+  // never drops failing tasks
+  defaultExponentialBackOff: exponentialBackOff(100, {
+    maxDistance: 3_000,
+  }),
+  // drops failing tasks after few retries
+  defaultExponentialBackOffAndDrop: exponentialBackOff(100, {
```

Comment:
I am not sure if there can be two defaults. Maybe just `backOffAndDrop`? Or maybe we should differentiate and have the functions be `camelCase` and values be `CONST_CASE`

Reply:
After some more thoughts:

in TaskQueue `const DEFAULT_RETRY = Retires.expo...`
in `/core/transaction-count/utils.ts` `export const BACK_OFF_AND_DROP = Retires...`